src/deluge/gui/l10n/g_seven_segment.cpp

```diff
@@ -135,6 +135,7 @@ PLACE_SDRAM_DATA Language seven_segment{
         {STRING_FOR_ANALOG_SQUARE, "ASQUARE"},
         {STRING_FOR_ANALOG_SAW, "ASAW"},
         {STRING_FOR_SAMPLE, "SAMP"},
+        {STRING_FOR_DX7, "DX7"},
```

Comment:
Looks like the source `.json` for these didn't get checked in, I still need to fix the genfile check...